src/main/resources/assets/alexscaves/lang/zh_cn.json

```diff
@@ -600,555 +601,556 @@
-  "alexscaves.sound.subtitle.darkness_incarnate_idle": "黑暗化身：颤抖",
+  "alexscaves.sound.subtitle.galena_gauntlet_use_loop": "方铅石护手:嗡嗡作响",
+  "alexscaves.sound.subtitle.galena_gauntlet_stop": "方铅石护手:电源关闭",
+  "alexscaves.sound.subtitle.seeking_arrow_hit": "追踪箭命中",
+  "alexscaves.sound.subtitle.seeking_arrow_lockon": "追踪箭锁定",
+  "alexscaves.sound.subtitle.resistor_shield_azure_loop": "电阻盾牌:震荡",
+  "alexscaves.sound.subtitle.resistor_shield_scarlet_loop": "电阻盾牌:震荡",
+  "alexscaves.sound.subtitle.resistor_shield_slam": "电阻盾牌:碰撞",
+  "alexscaves.sound.subtitle.resistor_shield_spin": "电阻盾牌:旋转",
+  "alexscaves.sound.subtitle.limestone_spear_throw": "石灰岩矛:抛出",
+  "alexscaves.sound.subtitle.limestone_spear_hit": "石灰岩矛:刺伤",
+  "alexscaves.sound.subtitle.primitive_club_hit": "原始大棒:重击",
+  "alexscaves.sound.subtitle.primitive_club_miss": "原始大棒:挥空",
+  "alexscaves.sound.subtitle.raygun_loop": "镭射枪:运作",
+  "alexscaves.sound.subtitle.raygun_reload": "镭射枪:填充",
+  "alexscaves.sound.subtitle.raygun_empty": "镭射枪:空",
+  "alexscaves.sound.subtitle.raygun_start": "镭射枪:激活",
+  "alexscaves.sound.subtitle.magic_conch": "神奇海螺:向深渊呼唤",
+  "alexscaves.sound.subtitle.magic_conch_summon": "深潜者们回应了向深渊的呼唤",
+  "alexscaves.sound.subtitle.sea_staff_cast": "水弹:被召唤",
+  "alexscaves.sound.subtitle.sea_staff": "水弹:出现",
+  "alexscaves.sound.subtitle.sea_staff_hit": "水弹:击中",
+  "alexscaves.sound.subtitle.sea_staff_bubble": "水弹:气泡声",
+  "alexscaves.sound.subtitle.ortholance_wave": "破浪长枪:呼啸",
+  "alexscaves.sound.subtitle.desolate_dagger_summon": "荒凉匕首:生成",
+  "alexscaves.sound.subtitle.desolate_dagger_hit": "荒凉匕首:击中",
+  "alexscaves.sound.subtitle.totem_of_possession": "支配图腾:激活",
+  "alexscaves.sound.subtitle.dreadbow_release": "恐惧弓:射击",
+  "alexscaves.sound.subtitle.dreadbow_rain": "黑暗箭雨落下",
+  "alexscaves.sound.subtitle.nuclear_explosion": "核爆炸",
+  "alexscaves.sound.subtitle.large_nuclear_explosion": "剧烈的核爆炸",
+  "alexscaves.sound.subtitle.nuclear_explosion_rumble": "地面震动",
+  "alexscaves.sound.subtitle.nuclear_explosion_ringing": "耳鸣",
+  "alexscaves.sound.subtitle.submarine_place": "潜艇:被放置",
+  "alexscaves.sound.subtitle.submarine_hit": "潜艇:受损",
+  "alexscaves.sound.subtitle.submarine_creak": "潜艇:开裂",
+  "alexscaves.sound.subtitle.submarine_repair": "潜艇:被修复",
+  "alexscaves.sound.subtitle.submarine_destroy": "潜艇:被破坏",
+  "alexscaves.sound.subtitle.submarine_light_on": "潜艇远光灯:打开",
+  "alexscaves.sound.subtitle.submarine_light_off": "潜艇远光灯:关闭",
+  "alexscaves.sound.subtitle.submarine_move_loop": "潜艇:螺旋桨声",
+  "alexscaves.sound.subtitle.submarine_sonar": "检测到敌对生命体",
+  "alexscaves.sound.subtitle.dark_cloud_appear": "黑暗实体出现",
+  "alexscaves.sound.subtitle.dark_cloud_disappear": "黑暗实体消失",
+  "alexscaves.sound.subtitle.dark_cloud_idle": "黑暗实体:尖叫",
+  "alexscaves.sound.subtitle.darkness_incarnate_enter": "黑暗化身:进入",
+  "alexscaves.sound.subtitle.darkness_incarnate_exit": "黑暗化身:离开",
+  "alexscaves.sound.subtitle.darkness_incarnate_idle": "黑暗化身:颤抖",

```

Comment:
All subtitle translations from line 831 to line 1152 don't match the existing format "<Sound source>：<Sound type>". They all need to be changed.